ARCHITECTURE.md

```diff
@@ -27,49 +27,66 @@ AI Town is split into a few layers:
-for the async work, and `convex/game/agents.ts` for the game loop logic.
+for the async work, and `convex/aiTown/agent.ts` for the game loop logic.
 If you would like to add new gameplay elements (that both humans and agents can interact with), add
-the feature to `convex/game`, render it in the UI in `src/`, and respond to it in `game/agent.ts`.
+the feature to `convex/aiTown`, render it in the UI in `src/`, and respond to it in `convex/aiTown/agent.ts`.
 
 If you have parts of your game that are more latency sensitive, you can move them out of engine
 into regular Convex tables, queries, and mutations, only logging key bits into game state. See
 "Message data model" below for an example.
 
-## AI Town game logic (`convex/game`)
+## AI Town game logic (`convex/aiTown`)
 
 ### Data model
 
 AI Town's data model has a few concepts:
 
-- Players (`convex/game/players.ts`) are the core characters in the game and stored in the `players` table.
-  Players have human readable names and descriptions, and they may be associated with a human user.
-  At any point in time, a player may be pathfinding towards some destination and has a current location.
-- Locations (`convex/game/locations.ts`) keep track of a player's position, orientation, and velocity
-  in the `locations` table. We store the orientation as a normalized vector.
-- Conversations (`convex/game/conversations.ts`) are created by a player and end at some point in time.
-- Conversation memberships (`convex/game/conversationMembers.ts`) indicate that a player is a member
-  of a conversation. Players may only be in one active conversation at any point in time, and conversations
-  currently have exactly two members. Memberships may be in one of four states:
+- Worlds (`convex/aiTown/world.ts`) represent a map with many players interacting together.
+- Players (`convex/aiTown/player.ts`) are the core characters in the game. Players have human readable names and
+  descriptions, and they may be associated with a human user. At any point in time, a player may be pathfinding
+  towards some destination and has a current location.
+- Conversations (`convex/aiTown/conversations.ts`) are created by a player and end at some point in time.
+- Conversation memberships (`convex/aiTown/conversationMembership.ts`) indicate that a player is a member
+  of a conversation. Players may only be in one conversation at any point in time, and conversations
+  currently have exactly two members. Memberships may be in one of three states:
   - `invited`: The player has been invited to the conversation but hasn't accepted yet.
   - `walkingOver`: The player has accepted the invite to the conversation but is too far away to talk. The
     player will automatically join the conversation when they get close enough.
   - `participating`: The player is actively participating in the conversation.
-  - `left`: The player has left the conversation, and we keep the row around for historical queries.
```

Comment:
interesting - we don't have this and instead just archive it?

Reply:
yep, exactly!